src/Collection/Functions/Result/DbalExpressionResult.php

```diff
@@ -56,6 +59,8 @@ public function __construct(
 		public readonly ?PropertyMetadata $propertyMetadata = null,
 		?callable $valueNormalizer = null,
 		public readonly string|array|null $dbalModifier = null,
+		public readonly ?ExpressionContext $context = null,
```

Comment:
nit: added mistakenly, not needed; or is havignExpression === null a good detection of having expression? 🤔